agent/csi/plugin/plugin.go

```diff
@@ -290,6 +290,9 @@ func (np *nodePlugin) NodePublishVolume(ctx context.Context, req *api.VolumeAssi
 	if len(req.VolumeID) == 0 {
 		return status.Error(codes.InvalidArgument, "Volume ID missing in request")
 	}
+	if req.AccessMode == nil {
+		return status.Error(codes.InvalidArgument, "AccessMode missing in request")
+	}
```

Comment:
Let's factor this out into a common helper and use it in the Stage/Unstage methods as well (every method that calls `makeCapability`). I'd also like to see that change in a separate commit.